[Fix][connector-http] fix when post have param (#8434)
apache/seatunnel: docs/en/connector-v2/source/Http.md

| connect_timeout_ms          | Int     | No       | 12000   | Connection timeout setting, default 12s.                                                                                                                                          |
| socket_timeout_ms           | Int     | No       | 60000   | Socket timeout setting, default 60s.                                                                                                                                              |
| common-options              |         | No       | -       | Source plugin common parameters, please refer to [Source Common Options](../source-common-options.md) for details                                                                 |
| keep_params_as_form         |    Boolean     | No       | false       | Whether the params are submitted according to the form, used for compatibility with legacy behaviors. When true, the value of the params parameter is submitted through the form. |

> | keep_page_param_as_http_param         |    Boolean     | No       | false       | Whether to set the paging parameters to params.                                                                                                                                   |

Reviewer: ```suggestion
| keep_page_param_as_http_param         |    Boolean     | No       | false       | Whether to set the paging parameters to params. For compatibility with legacy behaviors.                                                                                                                                   |
```

| connect_timeout_ms          | Int     | No       | 12000   | Connection timeout setting, default 12s.                                                                                             |
| socket_timeout_ms           | Int     | No       | 60000   | Socket timeout setting, default 60s.                                                                                                 |
| common-options              |         | No       | -       | Source plugin common parameters, please refer to [Source Common Options](../source-common-options.md) for details                    |

> | keep_params_as_form         |         | No       | -       | Whether the params are submitted according to the form                                                                               |

Reviewer: ```suggestion
| keep_params_as_form         |    Boolean     | No       | false       | Whether the params are submitted according to the form, used for compatibility with legacy behaviors. When true, the value of the params parameter is submitted through the form.                                                                               |
```

|----------------------------------------------------------|
| {"code":  200, "data":  "get success", "success":  true} |

### keep_params_as_form
For compatibility with old versions of http.
When set to true,`<params>` and `<pageing>` will be submitted in the form.
When set to false，`<params>` will be added to the url path,and `<pageing>` will be added to the body

### params
By default, the parameters will be added to the url path.

> If you need to keep the old version,see keep_params_as_form.

Reviewer: ```suggestion
If you need to keep the old version behavior, please check keep_params_as_form.
```

|----------------------------------------------------------|
| {"code":  200, "data":  "get success", "success":  true} |

### keep_params_as_form
For compatibility with old versions of http.
When set to true,`<params>` and `<pageing>` will be submitted in the form.

> When set to false，`<params>` will be added to the url path,and `<pageing>` will be added to the body

Reviewer: ```suggestion
When set to false，`<params>` will be added to the url path,and `<pageing>` will not be added to the body or form. It will replace placeholders in params and body.
```

```

### keep_params_as_form
When set to true,params and page params will be submitted in the form.
If the submission is application/json format,please do not set this value.

### params
By default, the parameters will be added to the url path.
If form submission is required,please set keep_params_as_form true.

> 

Reviewer: ```suggestion
```
